treeherder/config/urls.py

```diff
@@ -29,5 +29,7 @@
     from graphene_django.views import GraphQLView
     from treeherder.webapp.graphql.schema import schema
     urlpatterns += [
-        url(r'^graphql', csrf_exempt(GraphQLView.as_view(graphiql=True, schema=schema))),
+        url(r'^graphql',
```

Comment:
Would you mind filing a ticket to fix the regex so it doesn't match things like /graphqlzzzzz etc
Seems like the docs are wrong here:
http://docs.graphene-python.org/projects/django/en/latest/tutorial-plain/

I'm not sure if `r'^graphql$'` would work, or if that would stop query parameters (depending on if Django includes them in the string being compared).